content/en/docs/instrumentation/erlang/_index.md

```diff
@@ -15,4 +15,119 @@ Packages of the API, SDK and OTLP exporter are published to
 [opentelemetry](https://hex.pm/packages/opentelemetry) and
 [opentelemetry_exporter](https://hex.pm/packages/opentelemetry_exporter).
 
+## Version support
+
+OpenTelemetry Erlang supports Erlang 23+ and Elixir 1.13+.
+
+## Installation
```

Comment:
I see that you chose to pull out the "Installation" section from Getting started since you now have more content. In that case I'd like to suggest that you move this new content into a separate page.

For most languages, the landing page is quite short. We should aim for that here if possible.

@tsloughter @svrnm @cartermp et al. WDYT?

Reply:
I would generally agree, this probably belongs only in getting-started. I'm not opposed to having this here in theory, but if so, I'd like that to be true for all languages.

Reply:
Just pushed a fix that moves stuff that is needed to getting started. I simply removed the experimental apps stuff for now as that'll be added back when I do the metrics/logging docs updates soon.